zerver/lib/webhooks/common.py

```diff
@@ -20,5 +27,17 @@ def check_send_webhook_message(
     else:
         if user_specified_topic is not None:
             topic = user_specified_topic
-        check_send_stream_message(user_profile, request.client,
-                                  stream, topic, body)
+
+        try:
+            check_send_stream_message(user_profile, request.client,
+                                      stream, topic, body)
+        except JsonableError as err:
+            if str(err) == "Stream '{}' does not exist".format(stream):
```

Comment:
This check isn't great because of i18n.  See the subclasses of `JsonableError` for a better way to do this.